python/ray/air/session.py

```diff
@@ -275,6 +275,70 @@ def train_loop_per_worker():
     return session.local_rank
 
 
+@_warn_session_misuse(default_value=0)
+def get_local_world_size() -> int:
+    """Get the local rank of this worker (rank of the worker on its node).
+
+    .. code-block:: python
+
+        import time
+        from ray.air import session
+        from ray.air.config import ScalingConfig
+
+        def train_loop_per_worker():
+            if torch.cuda.is_available():
+                torch.cuda.set_device(session.get_local_rank())
```

Comment:
update this code snippet to show `get_local_world_size`?

Also let's rewrite this using the doctest syntax.

```
>>> import time
...
```